### DISCUSSION TITLE
Does Trivy scan YAML files?

### DISCUSSION THREAD

**Discussion by MusaddikAccuKnox** [aquasecurity/trivy#1332] [Q&A] [answered]
Hello, if I have Trivy Scan in my pipeline and I raise a PR including a YAML file in it. 
Will Trivy scan the YAML file for vulnerabilities?

**Comment by knqyf263** [comment, marked as answer]
Trivy currently supports Kubernetes YAML files. We're going to support CloudFormation files as well shortly.
https://aquasecurity.github.io/trivy/v0.20.2/misconfiguration/policy/builtin/

**Reply by MusaddikAccuKnox** [reply]
Thank you for replying. That clarifies my query!
